set.mm

```diff
@@ -550065,13 +550086,23 @@ A collection of theorems for commuting equalities (or
   $c EqvRels $. $( The class of equivalence relations $)
   $c EqvRel $. $( Equivalence relation predicate $)
 
-  $c ElEqvRel $. $( Elementhood equivalence relation predicate
-                    (used only for a placeholder definition) $)
+  $c CoElEqvRels $. $( The class of coelement equivalence relations $)
+  $c CoElEqvRel $. $( Coelement equivalence relation predicate $)
 
   $c Reds $. $( The class of redundants $)
```

Comment:
If possible, use a similar technique here, and localize constants introduced for your private usage at the moment.

Reply:
(same as in https://github.com/metamath/set.mm/pull/3301#discussion_r1267942266 )